.github/workflows/ci_dockers.yml

```diff
@@ -109,8 +109,10 @@ jobs:
             pytorch_version: 1.4
           - python_version: 3.7
             pytorch_version: 1.7
-          - python_version: 3.7
+          - python_version: 3.8
             pytorch_version: 1.8
+          #  - python_version: 3.9
```

Comment:
why not testing with it then?

Reply:
there was some problem in past, so I would rather leave it for separate PR